sensor_msgs/Image Table_image
---
bool dough_found
int32[] dough_center
int32[] dough_D1
int32[] dough_D2
int32 area
int32 radius_a
int32 radius_b